packages/evm/src/context.ts

```diff
@@ -190,22 +190,18 @@ export class EvmContext extends RelayerAbstract<ethers.ContractReceipt> {
     const amountBN = ethers.BigNumber.from(amount);
     const bridge = this.contracts.mustGetBridge(sendingChain);
 
-    let recipientAccount = recipientAddress;
-    // get token account for solana
-    if (recipientChainId === MAINNET_CHAINS.solana) {
-      let tokenId = token;
-      if (token === NATIVE) {
-        tokenId = {
-          address: await bridge.WETH(),
-          chain: sendingChainName,
-        };
-      }
-      recipientAccount = await this.wormhole.getSolanaRecipientAddress(
-        recipientChain,
-        tokenId as TokenId,
-        recipientAddress,
-      );
-    }
+    let tokenID: TokenId =
+      token === NATIVE
+        ? {
+            address: await bridge.WETH(),
+            chain: sendingChainName,
+          }
+        : token;
```

Comment:
should be const but also, this method of `native` to `TokenId` should be a util fn on the class if we need it in >1 spot